deployment/scripts/deploy-openshift.sh

```diff
@@ -309,7 +363,9 @@ kubectl wait --for=condition=Programmed gateway maas-default-gateway -n openshif
 echo ""
 echo "1️⃣1️⃣ Applying Gateway Policies..."
 cd "$PROJECT_ROOT"
-kustomize build deployment/base/policies | kubectl apply --server-side=true --force-conflicts -f -
+# Ensure MAAS_API_NAMESPACE is set with default (it should already be set above)
+export MAAS_API_NAMESPACE=${MAAS_API_NAMESPACE:-maas-api}
+kustomize build deployment/base/policies | envsubst '$MAAS_API_NAMESPACE' | kubectl apply --server-side=true --force-conflicts -f -
```

Comment:
Like here we really are super sure already :) Is there something messing up the value of the exported variable in the meantime?

Reply:
You never know when a bit flips half way thorugh a script :smile: 

This was an AI choice I should cleaned up but doing it now